.eslintrc.json

```diff
@@ -51,12 +61,12 @@
         "no-case-declarations": "error",
         "no-const-assign": "error",
         "no-constant-condition": "off",
-        "no-console": "warn",
+        "no-console": "off",
```

Comment:
I disabled it because it was useless noise during the update process. IMO, everything should be either error or off; warnings are suggestions and are easy to ignore when they seem to be unrelated noise. That's outside of the scope of this change though and I have reverted it back to `warn`.